src/cpp/RiderLink/Source/RD/src/rd_framework_cpp/src/main/task/WiredRdTaskImpl.h

```diff
@@ -47,15 +47,15 @@ class WiredRdTaskImpl : public RdReactiveBase
 		spdlog::get("logReceived")
 			->trace("call {} {} received response {} : {}", to_string(cutpoint->get_location()), to_string(rdid), to_string(rdid),
 				to_string(read_result));
-		scheduler->queue([this, read_result = std::move(read_result)]() mutable {
+		scheduler->queue([&, moved_result = std::move(read_result)]() mutable {
```

Comment:
Please, check this place.
There were conflicting commits:
1) https://github.com/JetBrains/UnrealLink/commit/6b02a078656ea6de44d9d85081150bca4759857c
2) https://jetbrains.team/p/net/repositories/UnrealLink/revision/31ede08fbe11744de4c38da44d73871c9ecea51b

Reply:
These change sets seem to be functionally equivalent. Let's go with the one from the external repo. So, yep, you did it correct.